browser/model/devstudio.js

```diff
@@ -9,14 +9,15 @@ import Logger from '../services/logger';
 import JdkInstall from './jdk-install';
 
 class DevstudioInstall extends InstallableItem {
-  constructor(installerDataSvc, targetFolderName, downloadUrl, fileName, devstudioSha256, additionalLocations, additionalIus) {
-    super(DevstudioInstall.KEY, downloadUrl, fileName, targetFolderName, installerDataSvc, true);
+  constructor(keyName, installerDataSvc, targetFolderName, downloadUrl, fileName, sha256sum, additionalLocations, additionalIus, useDownload) {
```

Comment:
Any particular reason to use keyName in constructor here, while other model classes use a combination of static Class.KEY and this.keyName?